addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -523,6 +524,22 @@ var FormController = BasicController.extend({
         var state = this.model.get(this.handle);
         this.renderer.confirmChange(state, state.id, [field]);
     },
+    /**
+     * @private
+     * @param {OdooEvent} ev
+     */
+    _onAttachClicked: function (ev) {
```

Comment:
this is kind of weird. I think that this should be done in the Attachment widget, if possible